File: C#/If Statements/Exercism - Darts.cs
// Link to the exercise: https://exercism.org/tracks/csharp/exercises/darts

using System;

public static class Darts
{
    public static int Score(double x, double y)
    {
        double _distance = Math.Sqrt(x * x + y * y);

        if (_distance <= 1)
            return 10;
        else if (_distance <= 5)
            return 5;
        else if (_distance <= 10)
            return 1;
        else
            return 0;
    }
}

File: Python/Comparisons/Exercism - Darts.py
# Link to the exercise: https://exercism.org/tracks/python/exercises/darts

from math import sqrt

def score(x: float, y: float) -> float:
    distance: float = sqrt(x**2 + y**2)
    
    if distance <= 1:
        return 10
    elif distance <= 5:
        return 5
    elif distance <= 10:
        return 1
    return 0